addons/mail/static/tests/qunit_suite_tests/components/message_tests.js

```diff
@@ -812,14 +796,12 @@ QUnit.test('data-oe-id & data-oe-model link redirection on click', async functio
 QUnit.test('chat with author should be opened after clicking on his avatar', async function (assert) {
     assert.expect(4);
 
-    this.data['res.partner'].records.push({ id: 10 });
-    this.data['res.users'].records.push({ partner_id: 10 });
-    const { createMessageComponent, messaging } = await start({
-        data: this.data,
-        hasChatWindow: true,
-    });
+    const pyEnv = await startServer();
+    const resPartnerId1 = pyEnv['res.partner'].create();
+    pyEnv['res.users'].create({ partner_id: resPartnerId1 });
+    const { createMessageComponent, messaging } = await start({ hasChatWindow: true });
     const message = messaging.models['Message'].create({
-        author: insert({ id: 10 }),
+        author: insert({ id: resPartnerId1 }),
```

Comment:
Should we can remove this `create` from messaging now ?

Reply:
There is multiple occurrences of this in this file.

Reply:
Yes but this is complicated, naive tests needs to be adapted to actually fetch data from server